### ISSUE TITLE
Prevent gitscope tool window from auto scrolling to top (enhancement)

### ISSUE THREAD

**Issue by srmarah** [comod/git-scope#11]
When opening files the first time in IntelliJ the gitscope tool window auto scrolls back up to the top. I've got a branch with 120+ files - I scroll down to the 80th file then double click on it the first time and the gitscope window auto scrolls back to the top. Then I scroll down to the 81st file, double click on it the first time and it auto scrolls to the top again, rinse and repeat. It would be nice if the gitscope tool window did not auto scroll when opening a file for the first time. Or maybe make it a settings toggle option. Thanks for the great plugin!

**Comment by ml-mave** [comment]
@comod @hoxi This is not yet fixed in the latest release; I can still reproduce this every time.

Just to recap:

* if file is already open and I click it -> no bug autoscroll
* if file wasn't already open and I click it -> bug autoscroll

I'm on Windows if that matters.

Still, thanks for the other fixes; the plugin is a lot better now.

**Comment by hoxi** [comment]
Yes, seems like this issue is still not fully fixed. I have a couple of usecases myself where I can reproduce the issue.

@comod we should reopen this issue, I will have a look at it again.
